### PR TITLE
Stories

### PR DESCRIPTION
**Pull request by TomPed**

### CODE REVIEW

**Comment by TomPed** [comment]
@stevepeak ready for a look whenever. With the stories you can:
### Check
- Instagram
- Twitter
- Github
- Domain
- Trademark
### Register
- Domain

File: brand-assember.story [hunk 1]
```diff
@@ -0,0 +1,32 @@
+http method:get path:'/assemble' as requset, response
```
Comment (stevepeak) [inline brand-assember.story]: `http-endpoint` now 👍

File: brand-assember.story [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      'message': 'Must provide: DomainName, Years, RegistrantFirstName, RegistrantLastName, RegistrantAddress1, RegistrantCity,
+                                RegistrantStateProvince, RegistrantPostalCode, RegistrantCountry, RegistrantPhone, RegistrantEmailAddress
+                                TechFirstName, TechLastName, TechAddress1, TechCity, TechStateProvince, TechPostalCode, TechCountry, TechPhone
+                                TechEmailAddress, AdminFirstName, AdminLastName, AdminAddress1, AdminCity, AdminStateProvince, AdminPostalCode
+                                AdminCountry, AdminPhone, AdminEmailAddress, AuxBillingFirstName, AuxBillingLastName, AuxBillingAddress1
+                                AuxBillingCity, AuxBillingStateProvince, AuxBillingPostalCode, AuxBillingCountry, AuxBillingPhone,
+                                AuxBillingEmailAddress, Extended attributes'
+    }
+  else
+    data = next './create/domain' args:req.body
```
Comment (stevepeak) [inline brand-assember.story]:
```diff
- data = next './create/domain' args:req.body
+ data = next `./create/domain` args:req.body
```
Need to use tick's for files 👍

File: check/domain.story [hunk 3]
```diff
@@ -0,0 +1,16 @@
+possible_domains = {}
+com_domain = namecheap check domain:brand_name + '.com'
```
Comment (stevepeak) [inline check/domain.story]:
```diff
- com_domain = namecheap check domain:brand_name + '.com'
+ com_domain = namecheap check domain:(brand_name + '.com')
or
+ com_domain = namecheap check domain:'{{brand_name}}.com'
```
Looks nicer huh?

**Comment by TomPed** [comment]
@stevepeak updated.

**Pull request merged**